grass7/imagery/i.landsat/i.landsat.qa/testsuite/test_i_landsat_qa.py

```diff
@@ -0,0 +1,63 @@
+"""Test i.landsat.qa
+
+(C) 2021 by the GRASS Development Team
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+
+@author Stefan Blumentrath
+
+Using examples from the manual
+i.landsat.qa --overwrite --verbose cloud="Medium,High" \
+    dataset="landsat_ot_c2_l2" \
+    output=./Cloud_Mask_rules.txt
+
```

Comment:
is this supposed to fail on purpose? Otherwise, it should be either cloud_confidence as fixed in the manual, or cloud="Yes"

Reply:
Thanks. This is now exactly as in the manual.